tests/test_geometry.py

```diff
@@ -493,6 +493,58 @@ def test_baran_tetrazole():
     assert atom_types == expected_types
 
 
+def test_aniline():
+    # although aniline is slightly pyramidal, we'd rather it
+    # be assigned planar for safety.
```

Comment:
Centroid angles are generally only computed for a G2_KINK -> G3_PLANAR case. This comment is stale (and was for when we also used centroid angles for G3_PYRAMIDAL -> G4_TETRAHEDRAL, which is unstable since the NH3 can invert, and G3_PLANAR -> G4_TETRAHEDRAL used cross-product based restraints. Now, both (G3_PYRAMIDAL, G3_PLANAR) will always use a cross product restraint.

Reply:
(removing this comment)